docs/manuals/characterization/stark_experiment.rst

```diff
@@ -0,0 +1,225 @@
+the qubit Stark shfit :math:`\delta f_s`.
+You can use a dedicated Stark drive channel if available.
+Otherwise, you may want to use a control channel associated with the physical
+drive port of the qubit.
+
+In a typical IBM device using the cross-resonance drive architecture,
+such channel can be identified with your backend as follows:
+
+.. jupyter-execute::
+
+    from qiskit.providers.fake_provider import FakeHanoiV2
+
+    backend = FakeHanoiV2()
+    qubit = 0
+
+    for qpair in backend.coupling_map:
+        if qpair[0] == qubit:
+            break
+
+    print(backend.control_channel(qpair)[0])
+
+This returns a control channel for which the qubit is the control qubit.
+This approach may not work for other device architectures.
+
+
+Characterizing Frequency Shift
+------------------------------
+
+One can experimentally measure :math:`\delta f_S` with the :class:`.StarkRamseyXY` experiment.
+Following pulse sequence illustrates how :math:`\delta f_S` is characterized
+by a variant of the Hahn-echo pulse sequence [5]_.
+
+.. jupyter-execute::
+    :hide-code:
+
+    %matplotlib inline
+
+    from qiskit_experiments.library import StarkRamseyXY
+    from qiskit import schedule, pulse
+    from qiskit.providers.fake_provider import FakeHanoi
+    from qiskit.visualization.pulse_v2 import IQXSimple
+
+    backend = FakeHanoi()
+    exp = StarkRamseyXY(
+        physical_qubits=[0],
+        backend=backend,
+        stark_amp=0.2,
+        delays=[100e-9],
+        stark_channel=pulse.ControlChannel(0),
+    )
+
+    circ = exp.circuits()[0]
+    ram_x_schedule = schedule(circ, backend=backend)
+
+    opt = {
+        "formatter.general.fig_chart_height": 10,
+        "formatter.margin.top": 0.1,
+        "formatter.margin.bottom": 0.2,
+        "formatter.label_offset.pulse_name": 0.1,
+        "formatter.text_size.annotate": 14,
+    }
+    ram_x_schedule.draw(time_range=(0, 1600), style=IQXSimple(**opt), backend=backend)
+
+The qubit is initialized in the :math:`Y`-eigenstate with the first half-pi pulse.
+This state may be visualized by a Bloch vector located on the equator of the Bloch sphere,
+which is highly sensitive to Z rotation arising from any qubit frequency offset.
+This operation is followed by a pi-pulse and another negative half-pi pulse
+right before the measurement tone filled in red.
+This sequence recovers the initial state when Z rotation is zero or :math:`\delta f_S=0`.
+
+As you may notice, this sequence is interleaved with two pulses labeled
+"StarkV" (Gaussian) and "StarkU" (GaussianSquare) filled in yellow, representing Stark tones.
+These pulses are designed to have the same maximum amplitude :math:`\Omega` resulting
+in the same :math:`\delta f_S` at this amplitude -- but why we need two pulses?
+
+Since :math:`\delta f_S` is amplitude dependent, Stark tones cause time-dependent
+frequency shift at pulse ramps. With a single Stark tone, you are only able to estimate
```

Comment:
```suggestion
Since :math:`\delta f_S` is amplitude dependent, the Stark pulses cause time-dependent
frequency shifts during the pulse ramps. With a single Stark tone, you are only able to estimate
```